charts/velero/templates/node-agent-daemonset.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: node-agent
+  name: {{ include "velero.fullname" . }}-node-agent
```

Comment:
https://github.com/vmware-tanzu/helm-charts/pull/599#issuecomment-2246745337

@Skaronator could you please check the Velero main code resolved this issue or not yet? Appreciated 🙏